Sources/CountryPicker/Config.swift

```diff
@@ -107,6 +108,9 @@ public struct Config: Configuration {
     /// sort by localized country names
     public var countriesSortingComparisonResult: ComparisonResult
 
+    /// display the close button as an 'X' image or as text.
+    public var showCloseButtonAsXImage: Bool
```

Comment:
let's use closeButtonStyle property instead of this boolean.
```
public enum DismissButtonStyle {
    case icon(image: UIImage)
    case title(title: String)
}
```